daemon/algod/api/algod.oas2.json

```diff
@@ -3681,8 +3681,7 @@
             "description": "The minimum transaction fee (not per byte) required for the\ntxn to validate for the current network protocol.",
             "type": "integer"
           }
```

Comment:
@algochoi Unrelated to the line:  Can you remove the following v1 related docs:  https://github.com/algorand/go-algorand/blob/master/daemon/algod/api/README.md#comments-below-are-for-v1-endpoints-and-are-deprecated?

Reply:
Sounds good - [b63973d](https://github.com/algorand/go-algorand/pull/4847/commits/b63973d37ff349c39db09e90146fc988f58ce726)

Reply:
@algochoi Resolving - Thanks for the README update.